tools/gulp/tasks/release.ts

```diff
@@ -51,27 +50,18 @@ task(':package:theming', [':bundle:theming-scss'], () => {
 });
 
 /** Bundles all scss requires for theming into a single scss file in the root of the package. */
-task(':bundle:theming-scss', execNodeTask(
-  'scss-bundle',
-  'scss-bundle', [
-    '-e', themingEntryPointPath,
-    '-d', themingBundlePath
-  ], {silentStdout: true}
-));
+task(':bundle:theming-scss', () => {
+  new Bundler().Bundle(themingEntryPointPath, [themingBundleDedupeGlob]).then(result => {
```

Comment:
Could you comment on what the arguments are here since it's not really clear from the call site?